docker/assertj/src/main/java/com/github/dockerjava/assertions/DockerJavaAssert.java

```diff
@@ -31,6 +35,12 @@ public ContainerAssert container(String name) {
       return new ContainerAssert(container);
    }
 
+   public CubeOutputAssert withContainer(String name) {
+         String[] cmd = {"bash", "-c", "for i in $(ps -axo pid --no-headers); do echo `ps -p $i -o comm=`; done"};
```

Comment:
So what we do then? Is it possible to parse the command just something simple like `containerString` or it is not possible?